rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/DiscoverEjbConfigurationXmlRuleProvider.java

```diff
@@ -58,8 +68,8 @@
  * @author <a href="mailto:bradsdavis@gmail.com">Brad Davis</a>
  * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
  */
-@RuleMetadata(phase = InitialAnalysisPhase.class, perform = "Discover EJB-JAR XML Files")
-public class DiscoverEjbConfigurationXmlRuleProvider extends IteratingRuleProvider<XmlFileModel>
+@RuleMetadata(phase = InitialAnalysisPhase.class, perform = "Discover EJB-JAR XML Files and other EJBs")
```

Comment:
It seems to me that build failures might be caused by this rule being executed not after [`AnalyzeJavaFilesRuleProvider`](https://github.com/windup/windup/blob/master/rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/ast/AnalyzeJavaFilesRuleProvider.java) so maybe adding `after = AnalyzeJavaFilesRuleProvider.class` could enhance this rule's behaviour.